modules/page/html/render/message.js

```diff
@@ -229,15 +249,24 @@ exports.create = function (api) {
     api.sbot.async.get({ id, private: true, unbox }, (err, value) => {
       if (err) return cb(err)
       var msg = { key: id, value }
-      if (isBlog(msg)) {
-        Blog(api.sbot.obs.connection).async.get(msg, (err, result) => {
-          if (err) return cb(err)
-          msg.body = result.body
-          cb(null, msg)
+
+      var me = api.keys.sync.id()
+      onceTrue(api.sbot.obs.connection, sbot => {
+        sbot.patchwork.contacts.isBlocking({ source: me, dest: value.author }, (err, blocking) => {
+          if (err) throw err
+          if (blocking) {
+            cb(null, null)
```

Comment:
Would love a comment / explanation here as well (assuming other comments don't explain it).

Reply:
I think I looked up how to get the page to render the "Can't display message" page, and this was how to do it. Will look into and add a comment.

Reply:
Looks like this was marked "resolved" but I'd still love a comment here.